api/institutions/serializers.py

```diff
@@ -191,3 +191,39 @@ def get_absolute_url(self, obj):
                 'version': 'v2',
             },
         )
+
+
+class InstitutionUserMetricsSerializer(JSONAPISerializer):
+
+    class Meta:
+        type_ = 'institution-users'
+
+    id = IDField(read_only=True)
+    user_id = IDField(read_only=True)
```

Comment:
This doesn't look right. I think what you want here is for just one `IDField`, called, `id`, and have its source be `user_id`. There should be only one IDField per record, and the id should be of the thing that the record describes. Looking at the sample JSON, if you had 30 users in ASU, each of them would have an ID of 'asu'. Whereas we know that the institution should be ASU, since that will be the URL that you're accessing them from. Does that make sense?